scripts/lib/CIME/XML/grids.py

```diff
@@ -428,7 +430,10 @@ def _get_gridmaps_v2(self, component_grids):
                 if grid1_value != grid2_value and grid1_value != 'null' and grid2_value != 'null':
                     map_ = gridmaps[node.text]
                     if map_ == 'idmap':
-                        logger.warning("Warning: missing non-idmap %s for %s, %s and %s %s "
+                        if grid1_name == "ocn_grid" and grid1_value == atm_gridvalue:
```

Comment:
This avoids the warning: Warning: missing non-idmap ROF2OCN_LIQ_RMAPNAME for ocn_grid, 0.9x1.25 and rof_grid r05.  When the ocn grid matches the atm grid this file isn't used and there is no need for the warning.